### DISCUSSION TITLE
Vertical switch?

### DISCUSSION THREAD

**Discussion by nagyrobi** [HASwitchPlate/openHASP#98] [Ideas]
Can we have vertical switch object?
Like this:
![kép](https://user-images.githubusercontent.com/1550668/111592670-5dfd7500-87c9-11eb-9574-e3a0139657e8.png)

**Comment by fvanroie** [comment]
Some LVGL widgets support vertical orientation, but it seems that the [switch widget](https://docs.lvgl.io/latest/en/html/widgets/switch.html) does not support it.

**Reply by nagyrobi** [reply]
It would make sense for a 2 or 3-gang wall switch...

**Reply by fvanroie** [reply]
It seems more of a design question. Why not a horizontal switch?
Functionally they are the same.

**Reply by nagyrobi** [reply]
Of course they are the same functionally. It's a functional design question, there's a conception that certain lights are always switched in every room by the left gang, and certain others by the right gang. Now when you replace in some of the rooms the physical switches with Lanbons for example, the first thing causing confusion will be is that which gang would be the left one, and which one will be the right one, when they are not placed like they used to be on the physical switch.

**Reply by fvanroie** [reply]
OK I see...

 This will require a change in lvgl switch widget though. Hasp-lvgl uses the capabilities of the common lvgl widgets.
Extending the widgets is beyond the scope of this project.

**Reply by nagyrobi** [reply]
Low priority, of course...
I was hoping there's a rotation property or sonething I missed :(
